### DISCUSSION TITLE
请问我们公司有采购需求，除了线上购买的方式，怎么通过公对公的采购渠道完成购买？（May I ask if our company has procurement needs? Besides online purchase, how can we complete the purchase through the business-to-business procurement channel?）

### DISCUSSION THREAD

**Discussion by PhilipWang34** [dashingsoft/pyarmor#1783] [Q&A]
请问我们公司有采购需求，除了线上购买的方式，怎么通过公对公的采购渠道完成购买？（May I ask if our company has procurement needs? Besides online purchase, how can we complete the purchase through the business-to-business procurement channel?）

**Comment by jondy** [comment]
目前只有线上模式, 提供代开的普通增值税发票
